doc/MONITORING.md

```diff
@@ -0,0 +1,64 @@
+Montoring
+==========
+
+<p align="left";>
+	<a href="https://www.elastic.co" alt="elk">
+	    <img src="images/logos/zabbix_logo.png" align="center" alt="ELK logo" width="200px" />
+    </a>
+</p>
+
+**[ZABBIX](https://www.zabbix.com)** Zabbix is an open-source monitoring software tool for diverse IT components, including networks, servers, virtual machines and cloud services. Zabbix provides monitoring metrics, among others network utilization, CPU load and disk space consumption.
+ 
+## Zabbix components
+
+### Zabbix Server
+
+Zabbix server is the central process of Zabbix software.
+
+The server performs the polling and trapping of data, it calculates triggers, sends notifications to users. It is the central component to which Zabbix agents and proxies report data on availability and integrity of systems. The server can itself remotely check networked services (such as web servers and mail servers) using simple service checks.
+
+### Zabbix Agent
+
+Zabbix agent is deployed on a monitoring target to actively monitor local resources and applications (hard drives, memory, processor statistics etc).
+
+### Zabbix Web ( frontend )
+
+Zabbix web interface is a part of Zabbix software. It is used to manage resources under monitoring and view monitoring statistics.
+
+### Zabbix Proxy 
+
+Zabbix proxy is a process that may collect monitoring data from one or more monitored devices and send the information to the Zabbix server, essentially working on behalf of the server. All collected data is buffered locally and then transferred to the Zabbix server the proxy belongs to.
+
+## How to use
+
+Make sure to enable zabbix in the `values.yaml` file, then to access the front-end use the following command:
+
+```
+minikube service zabbix-web
```

Comment:
Is it not fadi-zabbix-web ? @AyadiAmen

Reply:
@AyadiAmen you miss my comment I think. Is it not fadi-zabbix-web ?

Reply:
@alexnuttinck sorry i didn't see this comment, no it's not fadi-zabbix-web, it's zabbix-web. the release name is not included in the service name.

Reply:
Ok thanks, I think it should be change in the next releases of zabbix in order to have the same format of service names.

Reply:
@alexnuttinck it's done, i added the release name to the service name and now it's `fadi-zabbix-web` .

Reply:
perfect!

Reply:
yes, the version should be 0.1.2.